content/en/docs/components/katib/experiment.md

```diff
@@ -783,6 +784,50 @@ For more information, check:
 - The DARTS example —
   [`darts-gpu.yaml`](https://github.com/kubeflow/katib/blob/master/examples/v1beta1/nas/darts-gpu.yaml).
 
+#### Population Based Training (PBT)
```

Comment:
Please can you also add `suggestion-pbt` image to the Katib Config doc: https://www.kubeflow.org/docs/components/katib/katib-config/#suggestion-settings.